lib/api_remote.ml

```diff
@@ -113,6 +100,72 @@ let get_permalink ~(ctx : Context.t) ~(req : Slack_t.get_permalink_req) =
     ~name:(sprintf "chat.getPermalink (%s, %s)" req.channel req.message_ts)
     ~ctx `GET api_path Slack_j.read_get_permalink_res
 
+let www_form_of_get_upload_url_ext (req : Slack_t.get_upload_url_ext_req) =
```

Comment:
is this function used elsewhere? i don't think it's worth having this function out of `get_upload_url_external`. Just declare the `fields` variable and inside of it.